src/v/redpanda/admin/api-doc/security.json

```diff
@@ -112,4 +145,293 @@
+                "description": "Conflict",
+                "schema": {
+                    "$ref": "#/definitions/rbac_error_body"
+                }
+            }
+        }
+    },
+    "delete": {
+        "tags": [
+            "Roles"
+        ],
+        "summary": "Remove a role by name",
+        "operationId": "delete_role",
+        "consumes": [
+            "application/json"
+        ],
+        "parameters": [
+            {
+                "name": "role",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            }
+        ],
+        "produces": ["application/json"],
+        "responses": {
+            "204": {
+                "description": ""
+            },
+            "404": {
+                "description": "Not Found",
+                "schema": {
+                    "$ref": "#/definitions/rbac_error_body"
+                }
+            }
+        }
+    }
+},
+"/v1/security/roles/{role}/members": {
+    "get": {
+        "tags": [
+            "Members"
+        ],
+        "summary": "List members assigned a given role",
+        "operationId": "list_role_members",
+        "consumes": [
+            "application/json"
+        ],
+        "parameters": [
+            {
+                "name": "role",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            }
+        ],
+        "produces": ["application/json"],
+        "responses": {
+            "200": {
+                "description": "OK",
+                "schema": {
+                    "$ref": "#/definitions/role_members_list"
+                }
+            },
+            "404": {
+                "description": "Not Found",
+                "schema": {
+                    "$ref": "#/definitions/rbac_error_body"
+                }
+            }
+        }
+    },
+    "post": {
+        "tags": [
+            "Members"
+        ],
+        "summary": "Update role membership",
+        "operationId": "update_role_members",
+        "consumes": [
+            "application/json"
+        ],
+        "parameters": [
+            {
+                "name": "role",
+                "in": "path",
+                "required": true,
+                "type": "string"
+            },
+            {
+                "name": "add",
+                "in": "body",
+                "required": false,
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/role_member"
+                    }
+                }
+            },
+            {
+                "name": "remove",
+                "in": "body",
+                "required": false,
+                "schema": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/role_member"
+                    }
+                }
+            }
```

Comment:
Probably needs wrapping in an object or something, can't have two body "params".

Reply:
done